_posts/2019-09-06-kubernetes-and-hokusai.md

```diff
@@ -0,0 +1,146 @@
+traffic patters, maintain a zero-downtime deployment, and more. Very cool.
+
+# What is Hokusai?
+
+When Artsy's Engineering team was contemplating a move to Kubernetes from Heroku, we had beef with a few things.
+
+For one, we wanted to be able to do a few core things simply and easily using the command line. While Kubernetes
+has robust CLI tooling using [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/), it's also very
+complex. We wanted to be able to quickly and easily do the things we were used to doing with Heroku; we preferred
+`heroku logs` to `kubectl logs [POD]` (where we would have to either look up or know the specific pod name we
+wanted, even though pods are being spun up and taken down all the time).
+
+[Helm](https://helm.sh), the de-facto package manager for Kubernetes, also didn't quite fit our needs. Helm is
+great for big, complex implementations with Kubernetes, and it's very useful for managing releases. Artsy
+Engineering wanted something that didn't involve quite as many complex charts, and we're not as concerned as some
+orgs with versioned releases since our focus is mostly on web apps.
+
+Basically, we wanted our commands to be application-level instead of pod- or node-level. We wanted a little more
+abstraction than was offered by `kubectl`, and a little less than Helm.
+
+And there was the issue of review apps. Review apps are basically standalone versions of an application that fall
+completely outside a normal production pipeline. They allow you to test big or scary changes in functionality
+without even putting them on a staging instance (which could affect other developers' work or be deployed
+accidentally).
+
+Kubernetes doesn't support review apps out of the box. There are some add-ons that offer them, but at the time
+Artsy was looking to switch, I don't think they existed or were widespread.
+
+Thus was born Hokusai: a tool that makes interacting with applications deployed on Kubernetes from the command line
+simple. Need logs? `hokusai production logs`. Or a review app? There are a
+[few steps involved](https://github.com/artsy/hokusai/blob/master/docs/Review_Apps.md), but you can have a
+fully-featured copy of your app up and running in a few minutes.
+
+The end of this post has a larger cheatsheet for handy Hokusai commands, but for now, let's talk about how you can
+use it yourself.
+
+# How can I set up Hokusai with my project?
+
+I should begin by noting that Hokusai is currently only meant to work with AWS - if your application is running on
+a different provider, you might have to hold off on Hokusai for now :( (or
+[open a PR in Hokusai](https://github.com/artsy/hokusai) yourself!)
+
+Installing hokusai is super easy! You can see full instructions in the README on
+[GitHub](https://github.com/artsy/hokusai), but if you're already set up with Python, pip, Docker, Docker Compose,
+and Git, you can do a quick install with Homebrew:
```

Comment:
Suggestion: "you can do a quick install of Hokusai packed by [PyInstaller](https://www.pyinstaller.org/) with Homebrew"